app/helpers/application_helper.rb

```diff
@@ -46,6 +46,17 @@ def error_summary(form_object = @form_object)
     )
   end
 
+  def govuk_error_summary(form_object = @form_object)
```

Comment:
Can go as well in another PR, but also realised the new helper is lacking the functionality to set the 'Error' prefix in the page title.

As per:
https://github.com/ministryofjustice/c100-application/blob/master/app/helpers/application_helper.rb#L26

Reply:
We will look into this again tomorrow, we realised it was missing after we realised there was not coverage on CI

Reply:
No worries! I think it may help with some failing tests, to use the same code we are using on CAS for this helper:
https://github.com/ministryofjustice/c100-application/blob/master/app/helpers/application_helper.rb#L26